airbyte-integrations/connectors/source-rd-station-marketing/source_rd_station_marketing/streams.py

```diff
@@ -0,0 +1,203 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+from abc import ABC
+from datetime import date
+from typing import Any, Iterable, Mapping, MutableMapping, Optional
+
+import pendulum
+import requests
+from airbyte_cdk.sources.streams.http import HttpStream
+
+
+class RDStationMarketingStream(HttpStream, ABC):
+    url_base = "https://api.rd.services"
+    primary_key = None
+    page = 2
```

Comment:
Any reason why page start at 2?

Reply:
It started at 2 because the first requisition wouldn't use the next_page_token function, and the API behavior is to return the first page if no page parameter is set. I improved next_page_token logic to start at 1.